index.html

```diff
@@ -1997,6 +1997,47 @@ <h5>Get checksum for video/audio stream</h5>
     </div>
     <!-- ends Get checksum for video/audio stream -->
 
+    <!-- Get checksum for all video/audio streams -->
+    <label class="recipe" for="get_streamhash">Get individual checksums for all video/audio streams ("Streamhash")</label>
+    <input type="checkbox" id="get_streamhash">
+    <div class="hiding">
+      <h5>Get individual checksums for all video/audio streams ("Streamhash")</h5>
+      <p><code>ffmpeg -i <em>input_file</em> -map 0 -f streamhash -hash md5 - -v quiet</code></p>
+      <p>
+      The outcome is very similar to that of "-f hash", except you get one hash
+      per-stream, instead of one (summary) hash.  Another benefit is that you
+      don't have to know which streams, or how many to expect in the source
+      file. This is very handy for hashing mixed born-digital material.
+      </p>
+      <p>
+      This script will perform a fixity check on all audio and video streams in
+      the file and return one hashcode for each one.  This is useful for e.g.
+      be able to change to container/codec format later on and validate it
+      matches the original source.
+      </p>
+      <p>
+      The output is formatted for easily processing it further in any kind of
+      programming/scripting language.
+      </p>
+      <dl>
+        <dt>ffmpeg</dt><dd>starts the command</dd>
+        <dt>-i <em>input_file</em></dt><dd>path, name and extension of the input file</dd>
+        <dt>-map 0</dt><dd>map ALL streams from input file to output. If you omit this, ffmpeg chooses only the first "best" (*) stream: 1 for audio, 1 for video (not all streams).</dd>
+        <dt>-f streamhash -hash md5</dt><dd>produce a checksum hash per-stream, and set the hash algorithm to md5. See the official <a href="https://www.ffmpeg.org/ffmpeg-formats.html#streamhash-1" target="_blank">documentation on streamhash</a> for other algorithms and more details.</dd>
+        <dt>-</dt><dd>FFmpeg syntax requires a specified output, and <code>-</code> is just a place holder. No file is actually created. Choose an output filename to write the hashcode lines into a textfile.</dd>
+        <dt>-v quiet</dt><dd>(Optional) Disables FFmpeg's processing output. With this option it's easier to see the text output of the hashes.</dd>
+      </dl>
+      <p>
```

Comment:
Is this one missing a closing tag?

Reply:
Indeed! Sorry.
Will run "[HTML tidy](http://www.html-tidy.org/)".